packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -199,6 +199,9 @@
             <p>{{ showSend.text }}</p>
           </template>
         </dt-button>
+
+        <!-- @slot Slot for sendButton picker -->
+        <slot name="sendButton" />
```

Comment:
shouldn't this slot wrap the entire existing send button?

Reply:
oh I was thinking of setting showSend to false, but this is **_much_** better